tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -259,6 +261,11 @@ export class WebGPUBackend extends KernelBackend {
       // Data is on the CPU.
       return info.values;
     }
+    if (this.dispatchNumberInEncoder !== 0) {
+      this.dispatchNumberInEncoder = 0;
+      this.computePass.endPass();
+      this.submitQueue();
```

Comment:
Could this submitQueue() be removed? So we could get the same behavior as old delayed mode when WEBGPU_BATCH_DISPATCHING_CALLS set to negative value.